contracts/Swap/SwapRouter.sol

```diff
@@ -592,11 +592,11 @@ contract SwapRouter is Ownable2StepUpgradeable, RouterHelper, IPancakeSwapV2Rout
         address[] calldata path,
         address to,
         uint256 deadline
-    ) external override ensure(deadline) ensurePath(path) returns (uint256[] memory amounts) {
+    ) external override ensure(deadline) ensurePath(path) returns (uint256 swapAmount) {
```

Comment:
Why did you change the returned value? out of curiosity... because now it seems incoherent with `swapExactTokensForBNB`, for example

Reply:
Because the swap for tokens at a supporting fee will only return the swapped amount instead of an array of all amounts(swapped between) as the output of these amounts can not be assumed in the start of the Swap.